recipes/lineax/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "lineax" %}
+{% set version = "0.0.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/lineax-{{ version }}.tar.gz
+  sha256: e68f1eba2f352122fdce9adc0556684f31eb8364b1a00acee484dd6e44a34e5e
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.9,<4.dev0
+    - hatchling
+    - pip
+  run:
+    - python >=3.9
```

While I don't think that is needed, it should be the same as above.

```suggestion
    - python >=3.9,<4.dev0
```

---

It doesn't hurt b/c I believe the upper pin is unnecessary. I've seen this bug in grayskull before BTW, but I was too lazy to report ;-p